docs/rfc/rfc-1.adoc

```diff
@@ -281,15 +281,22 @@ it.
 [[heartbeat]]
 === Heartbeats
 
+Governable Parameters:
+
+- `failed_heartbeat_sortition_timeout`: The amount of time an operator is
+  removed from the sortition pool after failing a heartbeat.
+- `heartbeat`: The number of group members required for a heartbeat to successful.
+
 To make sure that older wallets are still accessible for redemption, we need to
-perform heartbeats. The signing group signs each bitcoin block and then does _not_
-publish the result. If a signer suspects other signers are not online, they can
-issue an on-chain challenge to publish a specified signed bitcoin block (with a
-maximum block age). Since publishing this information costs the signers gas, the
-challenger must pay a deposit to be distributed to the signers if they pass.
-
-Any signer unable to publish the signed block within a specified amount of time
-will begin to be slashed and the challenger will be rewarded.
+perform heartbeats. The signing group signs every 40th ethereum block and then
```

Comment:
I think 40th block should also be a governable parameter. Every 40th block means that they are singing every 10 minutes. I think it could make sense to make this time a little longer in some scenarios.

Reply:
addressed in a090772